.github/workflows/ci.yml

```diff
@@ -39,7 +39,9 @@ jobs:
       - uses: julia-actions/cache@v1
       - uses: julia-actions/julia-buildpkg@latest
       - uses: julia-actions/julia-runtest@latest
-      - uses: julia-actions/julia-uploadcodecov@v0.1
+      - uses: codecov/codecov-action@v3
+        with:
+          files: lcov.info
```

Comment:
https://github.com/JuliaPhysics/Measurements.jl/actions/runs/4495381585/jobs/7908941020?pr=141#step:7:29
```
[2023-03-22T23:23:14.040Z] ['error'] None of the following appear to exist as files: lcov.info
```
Great.  No clue how this is supposed to be used then

Reply:
It turns out it was necessary to run [`julia-actions/julia-processcoverage`](https://github.com/julia-actions/julia-processcoverage) before `codecov/codecov-action`, the work of `julia-actions/julia-processcoverage` was done automatically by `julia-actions/julia-uploadcodecov`.